ci-operator/step-registry/cucushift/installer/check/capability/cucushift-installer-check-capability-ref.yaml

```diff
@@ -22,5 +22,8 @@ ref:
       default: ""
       documentation: |-
         Extends the set of optional capabilities beyond what is specified in `baselineCapabilitySet`.
+  dependencies:
+  - name: "release:latest"
+    env: RELEASE_IMAGE_LATEST
```

Comment:
I think it should be `RELEASE_IMAGE_LATEST_FROM_BUILD_FARM` instead of `RELEASE_IMAGE_LATEST`, `RELEASE_IMAGE_LATEST` would be the exact string from API commands.

BTW, suggest to add the following debug lines in command.sh:
```
echo "RELEASE_IMAGE_LATEST: ${RELEASE_IMAGE_LATEST}"
echo "RELEASE_IMAGE_LATEST_FROM_BUILD_FARM: ${RELEASE_IMAGE_LATEST_FROM_BUILD_FARM}"
```